packages/react-badge/src/components/Badge/Badge.types.ts

```diff
@@ -2,8 +2,8 @@ import * as React from 'react';
 import type { ComponentProps, ComponentState, ObjectShorthandProps } from '@fluentui/react-utilities';
 
 export type BadgeSlots = {
-  root: ObjectShorthandProps<React.HTMLAttributes<HTMLElement>>;
-  icon?: ObjectShorthandProps<React.HTMLAttributes<HTMLElement>>;
+  root: ObjectShorthandProps<React.HTMLAttributes<HTMLElement>, React.Ref<HTMLElement>>;
```

Comment:
Should the second generic param be defaulted to React.Ref<HTMLElement> to avoid having to add it in tis case?